vuepress/docs/next/tutorials/backend-developers/build-and-deploy.md

```diff
@@ -109,12 +109,12 @@ git push -u origin master
 
 10. Push the tags `git push --tags`
 
-11. Install the `entando-bundle` using `npm install -g  @entando/entando-bundle@6.3.0`
+11. Install the `entando-bundler` using `npm install -g  @entando/entando-bundler@6.3.0`
```

Comment:
Might want to mention the alternative which is ent bundler. The CLI is automatically installed if you follow the quickstart install and it uses a private copy of entando-bundler with the version matching the CLI.  That avoids the global linking which can break later.